### PR TITLE
New end credit

### PR DESCRIPTION
**Pull request by Bomba0713**
Create video folder, end credits video added

### CODE REVIEW

File: ProjectSettings/EditorBuildSettings.asset [hunk 1]
```diff
@@ -4,29 +4,5 @@
...
-  - enabled: 1
-    path: Assets/Scenes/levels (don't edit)/night.unity
-    guid: 1850203cc0f87bb1ca8cc69cb16f2bf2
-  - enabled: 1
-    path: Assets/Scenes/levels (don't edit)/afternoon.unity
-    guid: 6ea732993ba09af0698b4197a6062131
-  - enabled: 1
-    path: Assets/Scenes/levels (don't edit)/evening.unity
-    guid: 0ffc1cd5a381384ca9c8baa399eb4f69
+  m_Scenes: []
```
Comment (viilis) [inline ProjectSettings/EditorBuildSettings.asset:7]: Why build settings are removed?

File: Assets/Scenes/Outdoor scene.unity [hunk 2]
```diff
@@ -122,14 +122,61 @@ NavMeshSettings:
     debug:
       m_Flags: 0
   m_NavMeshData: {fileID: 0}
+<<<<<<< HEAD
```
Comment (viilis) [inline Assets/Scenes/Outdoor scene.unity:125]: Scene file seem to have Git merge markers left. This will break the scene file.

**Review by viilis — CHANGES_REQUESTED** [review]

File: ProjectSettings/EditorBuildSettings.asset (hunk 1 above)
Comment (Bomba0713) [reply ProjectSettings/EditorBuildSettings.asset:7]: I am not sure I think this happened last time as well... when I open my unity it automatically says i did something to the editor setting.

Comment (Bomba0713) [reply ProjectSettings/EditorBuildSettings.asset:7]: how should i fix that? I didn't get any conflicts when i try pushing and pull request though

**Pull request closed without merging**